<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <widget class="QLineEdit" name="LE_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_hrId">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="placeholderText">
    <string>hr_id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>HR Id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_fullName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Full name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Full name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>User type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_userType">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_branch">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_userStatus">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_modifyUser">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>User ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_userName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LB_userID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>user id</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
